setup.py

```diff
@@ -9,11 +9,11 @@
 
 '''
 
-requires = ['Sphinx>=1.0']
+requires = ['Sphinx>=1.3']
```

Comment:
This is enough to make pip do the right thing and not install the new version for old sphinx.